src/components/ethno-art/migrations/RenameSongAndLiteracyMaterialToEthnoArts.migration.ts

```diff
@@ -0,0 +1,39 @@
+import { node, relation } from 'cypher-query-builder';
+import { BaseMigration, Migration } from '../../../core';
+import { ACTIVE } from '../../../core/database/query';
+
+@Migration('2021-10-20T16:28:16')
+export class RenameSongAndLiteracyMaterialToEthnoArts extends BaseMigration {
```

Comment:
This looks good! I think you are missing the export in the migrations/index.ts file which connects it to the module